<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="mycomputer" failures="1" tests="5" name="All" time="62.888" errors="0" timestamp="2022-11-05T19:27:28 EET">
  <testcase classname="TestClass.ContactUsTest" name="testcontactus" time="11.283"/>
  <testcase classname="TestClass.HomePageTest" name="CheckHomePageCount" time="0.059">
    <failure type="java.lang.AssertionError" message="expected [7] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [7] but found [0]
at TestClass.HomePageTest.CheckHomePageCount(HomePageTest.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- CheckHomePageCount -->
  <testcase classname="TestClass.TestLogin" name="login" time="11.322"/>
  <testcase classname="TestClass.TestLogin" name="loginwithincorrectPassword" time="4.062"/>
  <testcase classname="TestClass.SearchTest" name="CheckAllItemsareinStock" time="3.848"/>
  <testcase classname="pom.Base" name="@BeforeTest SetUP" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest SetUP -->
  <testcase classname="pom.Base" name="@BeforeTest SetUP" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest SetUP -->
  <testcase classname="pom.Base" name="@AfterTest Teamdown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest Teamdown -->
  <testcase classname="pom.Base" name="@BeforeTest SetUP" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest SetUP -->
  <testcase classname="pom.Base" name="@AfterTest Teamdown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest Teamdown -->
  <testcase classname="pom.Base" name="@AfterTest Teamdown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest Teamdown -->
</testsuite> <!-- All -->
